addons/wp/stringtable.xml

```diff
@@ -657,6 +663,15 @@
         <Key ID="STR_Lambs_WP_Suppressed">
             <English>Suppressed</English>
         </Key>
+        <Key ID="STR_Lambs_WP_Enabled">
+            <English>Enabled</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Disabled">
+            <English>Disabled</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_OnlyIfUGL">
+            <English>Only if Units have UGL Flares</English>

```

Comment:
Only when units are equipped with an UGL and flares

Reply:
```suggestion
            <English>Only when units are equipped with an UGL and flares</English>

```